### DISCUSSION TITLE
Conflicting versions found for fabric-api-lookup-api-v1 in 1.16.5

### DISCUSSION THREAD

**Discussion by BradClc** [FabricMC/fabric-api#1604] [Player support] [answered]
I used fabric-api-0.37.1+1.16 in my Modpacks(Just for playing with friends, not for distribution and monetization). 
Everything was fine when I first entered the game.But when I tried to open the save a second time, an error occurred.
Conflicting versions found for fabric-api-lookup-api-v1: used 1.2.0+6aebe11a7d, also found 1.2.0+84310a227d
I checked the list for Fabric,only found fabric-api-lookup-api-v1@1.2.0+6aebe11a7d 
I do not know how to found conflicting version.Here's the crash report,hope that can get help.
[crash-2021-08-01_22.50.45-server.txt](https://github.com/FabricMC/fabric/files/6912923/crash-2021-08-01_22.50.45-server.txt)

**Comment by warjort** [comment, marked as answer]
Your crash report does not show what you describe. It shows a minecraft class suffering recursive death.

class_2985 is advancement/PlayerAdvancementTracker.mapping:CLASS net/minecraft/class_2985 net/minecraft/advancement/PlayerAdvancementTracker
class_167 is advancement/AdvancementProgress.mapping:CLASS net/minecraft/class_167 net/minecraft/advancement/AdvancementProgress

Which suggests you need this fix:
https://www.curseforge.com/minecraft/mc-mods/advancements-debug

**Reply by BradClc** [reply]
I am very grateful and surprised to hear from you so quickly. Your suggestion did help me solve the problem. The game is running normally. Thanks again.
But my description is not unfounded. Since I used an unofficial third-party launcher to enter the game, the launcher had a separate log, and when the game crashed it was the launcher that alerted me to the problem：

Conflicting versions found for fabric-api-lookup-api-v1: used 1.2.0+6aebe11a7d, also found 1.2.0+84310a227d

I'll attach the latest file of the launcher below. The warnings appear in lines 14 and 15, which may not be the cause of the crash as you said, but I just want you to know that the warnings are real.
[latest.log](https://github.com/FabricMC/fabric/files/6912999/latest.log)

**Comment by K0-RR** [comment]
This warning gets printed when some mod jar-in-jars fabric-api-lookup-api-v1, but in the older version than the one provided with your fAPI.

Some mods don't depend on the whole Fabric API package, they just need one or two modules from it so these mods pack required modules so that you don't need to download fAPI.
